nautobot/docs/release-notes/version-2.3.md

```diff
@@ -8,37 +8,53 @@ This document describes all new features and changes in Nautobot 2.3.
+
 #### Object Metadata Models ([#5663](https://github.com/nautobot/nautobot/issues/5663))
 
 Added [a set of functionality](../user-guide/platform-functionality/metadata.md) for defining and managing object metadata, that is to say, data _about_ the network data managed in Nautobot, such as data provenance, data ownership, and data classification. For more details, refer to the linked documentation.
 
-#### Saved Views
+#### Python 3.12 Support ([#5429](https://github.com/nautobot/nautobot/issues/5429))
+
+Nautobot now supports Python 3.12, and Python 3.12 is now the default Python version included in the `nautobot` Docker images.
+
+#### Saved Views ([#1758](https://github.com/nautobot/nautobot/issues/1758))
 
 Added the ability for users to save multiple configurations of list views (table columns, filtering, pagination and sorting) for ease of later use and reuse. Refer to the [Saved View](../user-guide/platform-functionality/savedview.md) documentation for more details and on how to use saved views.
 
-#### Device Modules ([#2101](https://github.com/nautobot/nautobot/issues/2101))
+#### Worker Status Page ([#5873](https://github.com/nautobot/nautobot/issues/5873))
 
-Added new models for `ModuleBay`, `Module`, `ModuleType`, and `ModuleBayTemplate` to support modeling line cards and other modular components of a device. These models allow you to define a hierarchy of module bays and modules within a device, and to assign components (such as interfaces, power ports, etc.) to specific modules.
+User accounts with the `is_staff` flag set can access a new worker status page at `/worker-status/` to view the status of the Celery worker(s) and the configured queues. The link to this page appears in the "User" dropdown at the bottom of the navigation menu, under the link to the "Profile" page. Use this page with caution as it runs a live query against the Celery worker(s) and may impact performance of your web service.
 
 ### Changed
 
-#### Dynamic Group Cache Changes
+#### Changed TreeManager Default Behavior ([#5786](https://github.com/nautobot/nautobot/issues/5786))
+
+The `TreeManager` class (used for tree-models such as Location, RackGroup, and TenantGroup) default behavior has changed from `with_tree_fields` to `without_tree_fields`. This should improve performance in many cases but may impact Apps or Jobs that were relying on the old default; such code should be updated to explicitly call `.with_tree_fields()` where appropriate.
+
+#### Consolidated Detail View Action Buttons with a Dropdown Menu ([#5871](https://github.com/nautobot/nautobot/issues/5871))
+
+Consolidate common standard action buttons (those at the top right) in all detail views with a bootstrap split button dropdown.
+
```

Comment:
If we're going to call this one out, we should probably also call out #5870 and #5869 as well, probably in a single section titled something like "UI Button Consolidation"

Reply:
```suggestion
#### Consolidated UI Buttons ([#5869](https://github.com/nautobot/nautobot/issues/5869), [#5870](https://github.com/nautobot/nautobot/issues/5870), [#5871](https://github.com/nautobot/nautobot/issues/5871))

Various button groups in the "object list" and "object detail" views have been consolidated following a common UI pattern of a single button for the most common action plus a popup menu for less common actions.
```

Reply:
Oops sorry just saw this and pushed up a new commit. Lmk if the new change is acceptable?

Reply:
I don't think users care that it's a `bootstrap split button dropdown` specifically, my reword above was an attempt to make it a little less jargony. LMK.